### DISCUSSION TITLE
TokenExpired:token expired 怎么解决？

### DISCUSSION THREAD

**Discussion by yj451549728** [AlistGo/alist#4466] [General]
今天早上发现alist挂载的阿里网盘open进不去了，提示：failed get parent list: failed get dir: failed get parent list: failed to list objs: TokenExpired:token expired；
然后我进管理页面重新获取token，保存的时候报错Failed init storage: TokenExpired:token expired。
请问这是怎么回事？

**Comment by ban011** [comment]
更新alist 到最新版就可以了

**Reply by yj451549728** [reply]
感谢大佬
